.flake8

```diff
@@ -1,8 +1,10 @@
 [flake8]
 ignore = 
   E203, E266, E501, W503, E402, E731, C901, B950, B011,
-  B014  // does not apply to Python 2
+  B014,  // does not apply to Python 2
+  N812,  // i don't care
+  N804   // conflicts with B902
```

Comment:
Could we add a link to https://pypi.org/project/pep8-naming/?

Would be easier to figure out what the codes mean. A quick note on their meaning wouldn't hurt either?!

Reply:
Yeah I will look into how to document this better.

Reply:
I updated all the docs but couldn't figure out how to truly reference lints by name